apps/etterlatte-beregning/src/main/kotlin/avkorting/Avkorting.kt

```diff
@@ -77,9 +81,10 @@ data class Avkorting(
         virkningstidspunkt: YearMonth,
         bruker: BrukerTokenInfo,
     ): Avkorting {
+        // TODO kreve at det er inneværende år?
```

Comment:
Eller i alle fall feile hvis vi har grunnlag for år (periode fom evt?) > virkningstidspunkt